hw/syn/tools/dc/constraints.sdc

```diff
@@ -0,0 +1,160 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Generic constraints file for simple testsynthesis flow
+# This held very simple for now and needs to be refined
+
+set DEFAULT_CLOCK_UNCERTAINTY 0.5
+
+# TODO: consider splitting this into per-IP .sdc files
```

Comment:
at minimum I would think about splitting the top-level timing script away from the per-block ones, as the top-level timing behaves quite differently than a default block build.

Reply:
Yeah this is the plan. Currently I do not yet have the scripting mechanisms in place to source a different `constraints.sdc` file per IP block, hence I have put everything into this one and distinguish by block name. I.e., for submodules I currently have a set of simpler, more generic constraints further below. 

Once I am finished with integrating into the dvsim regression tool (#1749) I am going to switch to per-block constraints. These are going to live within the individual IP folders.